doc/Command_Index_User.tex

```diff
@@ -790,16 +790,23 @@ \subsubsection{End of Line Behavior}
 \subsubsection{Hyphenation}
 
 \macroname{\textbackslash gresethyphen}{\{\#1\}}{gregoriotex-main.tex}
-Tell Gregorio\TeX\ to when to place a hyphen between
-syllables in polysyllabic words in a score.  This is done by overriding \texttt{maximumspacewithoutdash} so subsequent changes to this dimension will override this command.
+Tells Gregorio\TeX\ how to place a hyphen between syllables in polysyllabic words in a score.  This is done by overriding \texttt{maximumspacewithoutdash} so subsequent changes to this dimension will override this command.
```

Comment:
I realize that this is old code that you haven't actually changed, but it got me thinking.
Given that you've introduced a flag `\ifgre@forcehyphen` my naive expectation would be that this should be used here instead of altering `maximumspacewithoutdash`.

Perhaps your flag should be called `\ifgre@showhyphen` instead, in which case we could revamp the way this setting works so that it doesn't have to alter `maximumspacewithoutdash` (but that sort of optimization need not be part of this change).